### PR TITLE
Feat/faucet changes

### PR DESCRIPTION
**Pull request by elclandestin0**
- Mobile view
- Default to connected wallet when wallet is connected
- Change color of "Come back in hh:mm:ss after requesting"
- Only make API request when address is complete in `External address` (This will help us ping the API less)
- Get tx hash in order to navigate to testnet when claim is done

### CODE REVIEW

**Comment by Anton-Mushnin** [comment]
Branch has conflicts

**Comment by Anton-Mushnin** [comment]
Screenshot 2024-10-02 at 20 16 54
It's not like in the design

**Comment by Anton-Mushnin** [comment]
Screenshot 2024-10-02 at 20 17 16
It's not like in the design

**Comment by Anton-Mushnin** [comment]
Screenshot 2024-10-02 at 20 19 40
It's not like in the design

**Comment by Anton-Mushnin** [comment]
Link from the notification center opens wrong explorer

**Comment by Anton-Mushnin** [comment]
Request button isn't disabled when address isn't valid

**Comment by Anton-Mushnin** [comment]
wrong placeholder
Screenshot 2024-10-04 at 11 32 43

**Comment by Anton-Mushnin** [comment]
[@Ishkhan](https://g7dao.slack.com/team/U071A1JH933)
, what action should be triggered when the floating notification is clicked?
[@Anton](https://g7dao.slack.com/team/U071GM3L8JC)
 the notification list should open below the Notifications button (edited)
 
 [https://g7dao.slack.com/archives/C073H2C4BMK/p1728031153577929?thread_ts=1727795961.178169&cid=C073H2C4BMK](url)

File: webapps/world-builder-dashboard/src/components/commonComponents/accountSelector/AccountSelector.tsx [hunk 1]
```diff
@@ -25,7 +21,7 @@ const ValueSelector = ({ values, onChange, selectedValue }: ValueSelectorProps)
       store={combobox}
       variant='unstyled'
       onOptionSubmit={(val: string) => {
-        const newSelection = values.find((n) => String(n.valueId) === val)
+        const newSelection = values.find((n) => String(n) === val)
```
Comment (Anton-Mushnin) [inline webapps/world-builder-dashboard/src/components/commonComponents/accountSelector/AccountSelector.tsx:24]: why not

 onOptionSubmit={onChange}

Comment (elclandestin0) [reply webapps/world-builder-dashboard/src/components/commonComponents/accountSelector/AccountSelector.tsx:24]: Because combobox doesn't like it!

**Pull request merged**